source/includes/_objects.md

```diff
@@ -327,150 +387,103 @@ if you are holding down the key.
-    var item = Inventory.getHeldItem();
+  var item = Player.getHeldItem();
 
-    if (!item.isEmpty()) {
-        var durabilityPercentage = Math.ceil(item.getDurability() / item.getMaxDurability() * 100);
+  if (item.getName() !== "tile.air.name") {
+    var durabilityPercentage = Math.ceil(
+      (item.getDurability() / item.getMaxDamage()) * 100
+    );
 
-        // If NaN, that means it's a block
-        if (isNaN(durabilityPercentage)) durabilityPercentage = "N/A (not a tool!)"
+    // If NaN, that means it's a block
+    if (isNaN(durabilityPercentage)) durabilityPercentage = "N/A (not a tool!)";
 
-        ChatLib.chat("Item: " + item.getDisplayName());
-        ChatLib.chat("Durability: " + durabilityPercentage + "%");
-        ChatLib.chat("Stack Size: " + item.getStackSize());
-    } else {
-        ChatLib.chat("&4You aren't holding anything!")
-    }
+    ChatLib.chat("Item: " + item.getName());
+    ChatLib.chat("Durability: " + durabilityPercentage);
+    ChatLib.chat("Stack Size: " + item.getStackSize());
+  } else {
+    ChatLib.chat("&4You aren't holding anything!");
+  }
 }
 ```
 
-All of the Inventory methods are shows below. Currently there is only one, but there may be more added in the future.
-
-Method Name | Description | Example return value
-------------|-------------|---------------------
-getHeldItem() | The player's currently held item | `InventorySlot.hotbar3`
-
-All of the InventorySlots share the same methods, which are listed below.
-
-Method Name | Description | Example return value
-------------|-------------|---------------------
-isEmpty() | Whether or not that inventory slot is empty | true, false
-isBlock() | Whether or not that inventory slot contains a block, as opposed to, say, a tool | true, false
-getDisplayName() | The item's display name | "Oak Fence", "Flint and Steel"
-getRegistryName() | The item's registry name | "fence", "flint_and_steel"
-getMaxDurability() | The item's max durability | 364, null (for blocks)
-getDurability() | The item's current durability | 115, null (for blocks)
-getStackSize() | The item's stack size | 1, 64
-getMetadata() | The item's metadata | 7
+In this case, we get the held item of the player. If the item isn't air, then it finds how damaged it is. If
+the durability isn't a number, that means the item has to be a block. Then, if the player isn't holding anything, it
+runs the last piece of the code, which is when the hand is empty.
+If you want to get the player's inventory as a list of `Items`, use `Player.getInventory()`.
 
 # XMLHttpRequests
 
-The XMLHttpRequests object has been removed in favor of the simpler, promise like [request](https://www.chattriggers.com/modules/v/request) module, its documentation can be found once downloaded.
\ No newline at end of file
+The XMLHttpRequests object has been removed in favor of the simpler, promise like [request](https://www.chattriggers.com/modules/v/request) module, its documentation can be found once downloaded.
```

Comment:
Let's just remove this section all together as this was removed a while ago. If someone needs to know how to do http requests, they can ask in Discord. It might also be a good idea to have a section of the slate dedicated to common operations like http requests, but that would be for another PR.